hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/ScaleUpTest.java

```diff
@@ -122,7 +122,6 @@ public void when_manyJobs() {
         }
         logger.info(jobs.size() + " jobs are running, adding a member");
         createJetMember(config);
-        sleepSeconds(2);
```

Comment:
If I understand the test correctly it has set delay for 1 second. It means that removing this `sleepSeconds(2);` will basically check that jobs are still running before cluster is scaled up. I think original test tried to check that they are running after scaling up.